tests/ui/suggestions/crate-or-module-typo.stderr

```diff
@@ -30,17 +30,23 @@ help: there is a crate or module with a similar name
    |
 LL |     bar: std::cell::Cell<bool>
    |          ~~~
+help: consider importing one of these items
+   |
+LL + use core::cell;
+   |
+LL + use std::cell;
+   |
+help: if you import `cell`, refer to it directly
+   |
+LL -     bar: st::cell::Cell<bool>
+LL +     bar: cell::Cell<bool>
+   |
```

Comment:
I'm noticing that these suggestions are now redundant. Could we check if the first suggestion has been emitted, and if so not try the import one?

Reply:
It's a little bit redundant, but sometimes the import one is the right advice to fix an error, I changed the code to skip the import suggestions if typo is found, and found this case changed:

```console
error[E0433]: failed to resolve: use of undeclared type `FromIterator`
  --> $DIR/suggest-tryinto-edition-change.rs:23:22
   |
LL |     let _v: Vec<_> = FromIterator::from_iter(&[1]);
   |                      ^^^^^^^^^^^^ use of undeclared type `FromIterator`
   |
   = note: 'std::iter::FromIterator' is included in the prelude starting in Edition 2021
   = note: 'core::iter::FromIterator' is included in the prelude starting in Edition 2021
help: a trait with a similar name exists
   |
LL |     let _v: Vec<_> = IntoIterator::from_iter(&[1]);
   |                      ~~~~~~~~~~~~
help: consider importing one of these items
   |
LL + use core::iter::FromIterator;
   |
LL + use std::iter::FromIterator;
   |
```

it's better than the new result after code changed:

```console
error[E0433]: failed to resolve: use of undeclared type `FromIterator`
  --> $DIR/suggest-tryinto-edition-change.rs:23:22
   |
LL |     let _v: Vec<_> = FromIterator::from_iter(&[1]);
   |                      ^^^^^^^^^^^^
   |                      |
   |                      use of undeclared type `FromIterator`
   |                      help: a trait with a similar name exists: `IntoIterator`
```

we are unsure which is better in all scenarios, so let's keep it right now.